tests/transports/test_urllib3.py

```diff
@@ -188,6 +188,16 @@ def test_ssl_verify_disable(waiting_httpsserver):
         transport.close()
 
 
+def test_ssl_verify_disable_http(waiting_httpserver):
```

Comment:
While we're at it, we could also add a test for HTTP and `server_cert=os.path.join(os.path.dirname(__file__), "wrong_cert.pem")`, since you fixed that bug too :+1:

Reply:
Done.